### ISSUE TITLE
Crashes

### ISSUE THREAD

**Issue by Luckyyang24** [iLLeniumStudios/fivem-appearance#165]
When I switch between job outfits then it's fine but as soon as I switch back to citizen outfit, my FiveM crashes all the time.

**Comment by TheiLLeniumStudios** [comment]
@Luckyyang24 Its because of your custom clothing. Nothing to do with fivem-appearance

**Comment by Luckyyang24** [comment]
> @Luckyyang24 Its because of your custom clothing. Nothing to do with fivem-appearance

I'm not running any custom clothing on my server though

**Comment by TheiLLeniumStudios** [comment]
> > @Luckyyang24 Its because of your custom clothing. Nothing to do with fivem-appearance
> 
> I'm not running any custom clothing on my server though

Post the crash logs then

**Comment by Luckyyang24** [comment]
![FiveM_b2699_DumpServer_D7aQqQrsJy](https://user-images.githubusercontent.com/106300955/191231818-a60cf5f1-0bd6-4dea-973c-33242bcd97e4.png)

**Comment by TheiLLeniumStudios** [comment]
@Luckyyang24 this is just a screenshot of the crash which is useless. Post the crash logs as I can't do anything with the above error

**Comment by Luckyyang24** [comment]
[CfxCrashDump_2022_09_20_10_11_12.zip](https://github.com/iLLeniumStudios/fivem-appearance/files/9606521/CfxCrashDump_2022_09_20_10_11_12.zip)

**Comment by Luckyyang24** [comment]
What's making me crash as I don't run custom clothing?

**Comment by TheiLLeniumStudios** [comment]
@Luckyyang24 The issue is with your player skins data. Empty players and playerskins table

**Comment by Luckyyang24** [comment]
> @Luckyyang24 The issue is with your player skins data. Empty players and playerskins table

but i only have like 3 stuff in there.
![heidisql_L6CZexCTCi](https://user-images.githubusercontent.com/106300955/191472208-9a3955b3-39fa-4ff7-ae42-489d81752600.png)
![heidisql_s1KP3fFGXb](https://user-images.githubusercontent.com/106300955/191472210-1477f962-4dbe-477f-903c-5653c8ca7f6e.png)

**Comment by TheiLLeniumStudios** [comment]
Yes that data doesn’t look correct. Delete your characters and create them again

**Comment by Luckyyang24** [comment]
Even after deleting the table and recreate, i still crash

[CitizenFX_log_2022-09-21T234021.log](https://github.com/iLLeniumStudios/fivem-appearance/files/9621035/CitizenFX_log_2022-09-21T234021.log)

**Comment by Luckyyang24** [comment]
![image](https://user-images.githubusercontent.com/106300955/191629045-5720a208-6c8a-4a37-bd42-428104739c26.png)
![image](https://user-images.githubusercontent.com/106300955/191629076-aa3df4d3-b9df-4aaa-a2b1-4a4ce2f57e67.png)

**Comment by TheiLLeniumStudios** [comment]
@Luckyyang24 the 1st one is problematic with the number. If you've deleted everything from your table when I asked you to and it still appeared then you haven't setup fivem-appearance correctly. I will suggest you to go through all the installation instructions again and make sure that everything is configured correctly

**Comment by tcantrell** [comment]
Similar crashes for me. If I cancel clothing purchase it will crash. Leads to issues of players crashing inside a mlo and will crash again on spawn last location. More then likely something I did, but I'll attach a crash report. 

[CitizenFX_log_2022-10-16T234539.log](https://github.com/iLLeniumStudios/fivem-appearance/files/9796029/CitizenFX_log_2022-10-16T234539.log)
